<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookCover</class>
 <widget class="QDialog" name="BookCover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>请选择《软件保护及破解技术——原理与实践》章节</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;本程式为&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://bbs.pediy.com/&quot;&gt;&lt;span style=&quot; font-size:16pt; text-decoration: underline; color:#0000ff;&quot;&gt;《软件保护与分析技术》&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://bbs.pediy.com/&quot;&gt;&lt;span style=&quot; font-size:16pt; text-decoration: underline; color:#0000ff;&quot;&gt;《代码虚拟与自动化分析》&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;随书演示程式&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;该程式仅用于配合理解书中内容，请勿用于其它任何用途&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rbk1">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>软件保护与分析技术</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labTip">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;第1章的标题为：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbText">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>打造函数通用追踪器</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>软件保护技术</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>打造进程拍照机</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>软件保护强化</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>打造资源重建工具</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WINLICENSE脱壳</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>打造伪调试器</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VMPROTECT虚拟机分析</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>软件破解技术</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>软件保护系统</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WINLICENSE虚拟机分析</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>打造函数监视器</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>软件破解工具</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>软件分析技巧</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VMPROTECT脱壳</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>打造预执行调试器</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>打造重定位修复工具</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="rbk2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>代码虚拟与自动化分析</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labTip2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;第1章的标题为：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbText2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>OP实时调试分析</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>自动化分析原理</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>代码虚拟机结构分析</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>实时监控分析</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>打造自动化分析工具</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Winlicense 2.3.2 分析</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OP分支功能性分析</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>花型替换分析</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>模拟虚拟化</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>xVMDebug插件</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>代码虚拟化原理</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>对比测试分析</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VMProtect分析</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnOK">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>继续</string>
       </property>
       <property name="icon">
        <iconset resource="xApiSpy.qrc">
         <normaloff>:/ico/images/Continue.ico</normaloff>:/ico/images/Continue.ico</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
       <property name="icon">
        <iconset resource="xApiSpy.qrc">
         <normaloff>:/ico/images/stop.ico</normaloff>:/ico/images/stop.ico</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="xApiSpy.qrc"/>
 </resources>
 <connections/>
</ui>
